app/models/support_interface/vendor_api_monitor.rb

```diff
@@ -29,6 +29,13 @@ def no_sync_in_24h
         .where("last_sync < now() - interval '24 hours' OR last_sync IS NULL").order('last_sync DESC')
     end
 
+    def no_sync_in_7d
```

Comment:
hmm what are your thoughts around making the sync weekly, I'm thinking we keep it the same for now as they should be syncing more regularly?

Reply:
As we will only be posting this once a week (and on a Monday) it's likely that it will mostly flag that providers have not synced in the last 24hours, which won't add much value. So if we do keep it then it would make sense to extend the time period we check against.

Reply:
ah good point. Yea lets keep to a week then, and if we miss something we'll amend again 👍